README.md

```diff
@@ -1,27 +1,31 @@
 # Discovery Bib/Item Poster
 
-Reads bibs or items from stream, posts to bib or item service.
+This lambda should be deployed to two different lambdas.
+One to handle bibs, another for items. It reads those bibs or items from stream, then posts them to the bib or item service.
 
 ## Setup
 
-```
+1.  ```
 npm install
 npm install -g node-lambda
-node-lambda setup
```

Comment:
Yup, I included `example.env` and `deploy.example.env` and added instructions about copying them instead of relying on the auto generation.